databases/functions/userFollowUser/user-block.js

```diff
@@ -26,7 +27,7 @@ module.exports = async (
   if (!userFollowUserHistoryModel)
     throw new Error('userFollowUserHistoryModel is required to block user');
 
-  const {postId = '', transaction = null} = params;
+  const {postId = '', commentId = '', transaction = null} = params;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `commentId` parameter has been added but it's not clear how it's being used in the function. If it's not used, consider removing it to avoid confusion. If it's intended for future use, add a comment explaining its purpose.
```diff
- const {postId = '', commentId = '', transaction = null} = params;
+ const {postId = '', transaction = null} = params; // if commentId is not used
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->